src/features/auth/signup/validation.ts

```diff
@@ -0,0 +1,7 @@
+//these are taken from the backend
+export const nameValidationPattern = /\S+/;
+export const usernameValidationPattern = /^[a-z][0-9a-z_]*[a-z0-9]$/;
+
+export function validateBirthdate(stringDate: string) {
+  return !isNaN(new Date(stringDate).getTime());
```

Comment:
```suggestion
  return !isNaN(new Date(stringDate).getTime()) || "Not a valid date.";
```
In order to enable the clean up in the other file.